content/articles/how-to-use-brotli-compression-method/index.md

```diff
@@ -0,0 +1,144 @@
+### Introduction
+Data compression is the process of encoding files to reduce their size, making them take up less space on your server and allowing them to be transferred to site clients more quickly. Compression reduces the amount of bandwith required for website rendering, which speeds up the process. tatic assets such as *.js*, *.css* and *.json* text files are commonly used in web application projects, with redundancy and compression playing an important role.
+
+### Prerequisites
+- Compression algorithms(Lossy & Lossless data compression)
+  
+### What is compression algoritm
+This is the process of reducing files in order to decrease their size and make them more portable.
+During the decompression process, it's also utilized to restore data to its original state.
+
+### What is Brotli
+Brotli is a lossless data compression algorithm that employs a variant of the *LZ77 algorithm*, as well as *Huffman coding* and *second-order context modelling*. Its compression ratio is comparable to the best general-purpose compression methods currently available.
+*LZ77 algorithm*,*Huffman coding* and *second-order context modelling* are lossless type of data compression algorithms.

```

Comment:
You haven't explained them here? I had asked you to add atleast 1 - 2 lines of explanations with references